src/main/webapp/js/notification.js

```diff
@@ -0,0 +1,16 @@
+function showNotification(message, options, reload = false) {
+    notificationBar.show(message, options);
+    if (reload) {
+        window.setTimeout(function () {
+            location.reload();
+        }, notificationBar.DELAY);
+    }
+}
+
+function showNotificationOK(message) {
+    showNotification(message, notificationBar.OK, true);
```

Comment:
There's a small problem with this behaviour. The page only refreshes when the success notification disappears. So for about 2 seconds, the notification shows but the content is not updated. With the alerts, as soon as the user clicked OK, the page was refreshed. Would it be possible to use a modal popup here? Also note that I'm working on #58 which will completely change the UI.

Reply:
Hi, thanks for your comment. I will search for modal solution, not sure though if its possible or not. For providing similar functionality I added optional reload feature, which waits for notification.DELAY.

About the UI change, of course It is up to you. We can apply this change until the new UI is published.

@timja Do you know any examples for Modal implementation ? Thanks.

Reply:
better to not refresh the page and to do a live update, or let the user refresh it themselves really.

Reply:
I think that is topic of the UI change that @AbhyudayaSharma mentioned. Should we decline this request then ?

Reply:
Yeah, the code will likely still be useful, thanks for doing this!

Reply:
No problem